UserSettings/EditorUserSettings.asset

```diff
@@ -5,6 +5,9 @@ EditorUserSettings:
   m_ObjectHideFlags: 0
```

Comment:
Diese Filme bitte nie commiten.
Eventuell sogar in die .gitignore mit aufnehmen.
Das sind Files die deine unity Editor Instanz betreffen.